fe2-android/app/build.gradle

```diff
@@ -16,179 +16,147 @@ android {
 
         testInstrumentationRunner "com.github.dedis.popstellar.HiltTestRunner"
 
+        // The following feature is not working properly with the coverage on the current build version
+        // As it's purpose is to clear physical data between each tests, its is not used yet.
```

Comment:
On a side note, isn't the Android crypto key manager stuff considered "physical data" ?

Reply:
I am not sure, but I don't think so as reinstalling the app does not seem to affect the device public key.